data/human/news.txt

```diff
@@ -2150,3 +2150,346 @@ news "graffiti on spaceport wall"
+		"pushed off the pad."
+
+news "Careless Landing"
+	location
+		planet "Jakobsen"
+	name
+		word
+			"Recklessness"
+	message
+		word
+			"A Sparrow"
+			"An Enforcer"
+			"A Shuttle"
+			"A Heavy Shuttle"
+			"A Star Barge"
+			"A Berserker"
+			"A Fury"
+			"A Hawk"
+			"A Wasp"
+		word
+			" lands on a pad meant for heavier ships. Air traffic control is immediately upon the captain to move ${Pronoun} ship. The captain "
+		phrase
+			"Careless Ends Well"
+			"Careless Ends Poorly"
+
+phrase "CCOR Song Description"
+	word
+		"a song about "
+	word
+		"community and everyone pitching in where they can"
+		"the tyranny of a man name Bailey, until he was overthrown"
+		"the CCOR, praising their accomplishments"
+		"resistance and opposition to the current planetary administration"
+
+phrase "Worker's Demands"
+	word
+		"safety regulations"
+		"increased pay"
+		"restoration of the CCOR"
+		"justice against planetary security brutality"
+		"immediate repositioning of the spaceport"
+
+news "Discontent"
+	location
+		planet "Bailey"
+	name
+		word
+			"Conflict"
+	message
+		word
+			"Allegations that planetary authorities will no longer move the spaceport has provoked "
+		word
+			"large scale "
+			"mass "
+			"moderate "
+			"scattered "
+		word
+			"riots. "
+			"protests. %"
+		word
+			"The rioters are "
+		word
+			"singing ${CCOR Song Description}. "
+			"chanting a list of their demands, including ${Worker's Demands}. "
+			"sizing up the planetary security forces. "
+		word
+			"Security eventually steps in, "
+		word
+			"but they're pushed back. "
+			"but they're routed. "
+			"and they disperse the demonstrators. "
+			"and the demonstrators scatter but don't appear dissuaded."
+		replace
+			"% The rioters" "The protesters"
+			"%" ""
+
+news "Clandestine"
+	location
+		planet "Alvorada"
+	name
+		word
+			"Observation"
+	message
+		word
+			"You can't escape the feeling of being watched. It"
+		word
+			"'s probably the security cameras; most of them are still running."
+			" feels like there are eyes around every corner."
+			"'s the pirates; a small number of them seem to be observing you."
+		word
+			" After some time of this "
+		word
+			"a silent alarm tells you that your should check on your ship. "
+			"it may be wise to make sure no one's tampering with your ship. "
+		word
+			"You find "
+		word
+			"a lone pirate prodding the ship with rudimentary tools. "
+			"a few pirates watching but not apparently touching it. "
+			"no one there. The area around your ship is almost suspiciously devoid of people. "
+		word
+			"Whatever is going on, your ship appears unscathed."
+
+news "Raiders returning to Mordente-Bridi"
+	location
+		planet "Mordente-Bridi"
+	name
+		word
+			"Repairs"
+	message
+		word
+			"Watching the microgravity section of the spaceport from the bar, you "
+		word
+			"observe "
+			"notice "
+			"spot "
+			"spy "
+		word
+			"a Bastion "
+			"a Firebird "
+			"a Falcon "
+			"a Leviathan "
```

Comment:
More ship ideas: 
Saber
Osprey
Bulwark?

Reply:
we need somthing like: Random(Ship::OfClass("Medium Warship"))->ModelName()

Reply:
That would be nice, but I’m not sure the CCOR would be using Splinters or Aeries.